issue-001/array_sum.py

```diff
@@ -0,0 +1,11 @@
+#!/bin/python3
+
+def array_sum(a):
+
+
+	arr_sum = 0
+
+	for i in arr:
+    	arr_sum += i
```

Comment:
This should be tabbed, and the variable here is `a` not `arr`:
```python
for i in a:
    arr_sum += i
```
Most likely you didn't test your code.